tests/conftest.py

```diff
@@ -7,6 +10,9 @@
 from tests.test_magic_link.magic_link_data import expected_magic_link_data
 from tests.test_magic_link.magic_link_data import expected_magic_link_response
 
+if platform.system() == "Darwin":
+    multiprocessing.set_start_method("fork")  # Required on macOSX
```

Comment:
shall we make a ticket to figure a way around this?

Reply:
it was working without this ( multiprocessing) line however since the pip-complie, it starts to fail so probably we should figure it out.

Reply:
Added JIRA ticket here: https://digital.dclg.gov.uk/jira/browse/FS-1168 and issue on this repo here https://github.com/communitiesuk/funding-service-design-notification/issues/12